src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Unix/DirectoryBasedStoreProvider.cs

```diff
@@ -448,7 +320,7 @@ private static void EnsureDirectoryPermissions(string path, uint userId)
                 throw new CryptographicException(SR.Format(SR.Cryptography_OwnerNotCurrentUser, path));
             }
 
-            if ((dirStat.Mode & (int)Interop.Sys.Permissions.Mask) != (int)Interop.Sys.Permissions.S_IRWXU)
+            if ((dirStat.Mode & (int)Interop.Sys.Permissions.S_IRWXU) != (int)Interop.Sys.Permissions.S_IRWXU)
```

Comment:
Just to be sure, the intent here is that we only care that you have at least RWX owner perms, and we don't care if there are any perms beyond that, right? (Whereas previously we did care if there were perms beyond that.)